apps/guide/src/components/Footer.tsx

```diff
@@ -45,7 +45,7 @@ export default function Footer() {
 							</div>
 						</div>
 						<div className="flex flex-col gap-2">
-							<div className="text-lg font-semibold">Project</div>
+							<div className="text-lg font-semibold">Projects</div>
```

Comment:
This is somewhat the same on the documentation, so if a change is made here, it should probably be made there too.